ci/values/k3s-plugin-server-split.yaml

```diff
@@ -36,3 +36,17 @@ plugins:
 # Enable pluginsAsync for this test
 pluginsAsync:
   enabled: true
+
+# Enable monitoring deployments. We are using these in our own deployments so
+# it's worth having these tested before they hit our clusters.
+loki:
+  enabled: true
+
+promtail:
+  enabled: true
+
+grafana:
+  enabled: true
+
+eventrouter:
+  enabled: true
```

Comment:
Should we add prometheus and few exporters as well?

Reply:
shoot, yes. I knew I was missing something, my mind was just on logs

Reply:
Plus I was thinking that maybe I should enable just for our live version, if only to avoid e.g. having to be blocked by e.g. a dependency not supporting a specific version.

Will follow up as appropriate.